### DISCUSSION TITLE
Add support for Blade template engine

### DISCUSSION THREAD

**Discussion by ahangarha** [wintercms/winter#184] [Ideas]
Support for Blade template engine was an old demand for OctoberCMS. But last discussion in their forum is dated almost 6 years ago. Things has changed since the and I wanted to see if there is any perspective for adding support for Blade template engine in WinterCMS?

**Comment by bennothommo** [comment]
Not at present. Twig has ample support for security and sandboxing, and has already been well established in Winter, so it's unlikely we'll switch to Blade.

It's a great candidate for a plugin though, and should a developer working on such a plugin need some support from the core codebase to make that happen, we'd be happy to make that so.
